pkgs/development/python-modules/attacut/default.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+
+  setuptools,
+
+  pytestCheckHook,
+
+  docopt,
+  fire,
+  numpy,
+  python-crfsuite,
+  pyyaml,
+  six,
+  ssg,
+  torch,
+}:
+
+buildPythonPackage {
+  pname = "attacut";
+  version = "1.1.0-dev";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "PyThaiNLP";
+    repo = "attacut";
+    rev = "eec888f6e06ab756ba4d4e0aa9261e949f3516dd";
```

```suggestion
    rev = "refs/tags/v${version}";
```
There is no significant difference between the latest release tag and HEAD.
I think you should adopt the tag for maintainability.
https://github.com/PyThaiNLP/attacut/compare/v1.1.0-dev...eec888f6e06ab756ba4d4e0aa9261e949f3516dd